src/Parser/Core/Modules/Items/GarothiFeedbackConduit.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import ITEMS from 'common/ITEMS';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from "../../../../common/SPELLS";
+import { formatThousands } from "../../../../common/format";
+
+const BASE_ITEM_LEVEL = 935;
+const BASE_HASTE_VALUE = 872;
+
+class GarothiFeedbackConduit extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+  currentILvl = 0;
+  baseILvl = 935;
+  staticHaste = 0;
+  lastProcc = null;
+  totalProccValue = [];
+  currentUptime = 0;
+  currentProccValue = 0;
+
+  totalAverageHasteGained = 0;
+  on_initialized() {
+    this.active = this.combatants.selected.hasTrinket(ITEMS.GAROTHI_FEEDBACK_CONDUIT.id);
+    if(this.active) {
+      this.currentILvl = this.combatants.selected.getTrinket(ITEMS.GAROTHI_FEEDBACK_CONDUIT.id).itemLevel;
+      this.staticHaste = Math.round(BASE_HASTE_VALUE* Math.pow(1.15,((this.currentILvl-BASE_ITEM_LEVEL)/15)) * Math.pow(0.994435486, (this.currentILvl-BASE_ITEM_LEVEL)));
```

Comment:
You should check out `common/stats/calculateSecondaryStatDefault`

Reply:
Oh, neat.